public/app/plugins/datasource/prometheus/datasource.ts

```diff
@@ -689,7 +697,7 @@ export class PrometheusDatasource extends DataSourceApi<PromQuery, PromOptions>
 
   async loadRules() {
     try {
-      const res = await this.metadataRequest('/api/v1/rules');
+      const res: any = await this.metadataRequest('/api/v1/rules');
```

Comment:
Changed to `any` here because of `const body = res.data || res.json();` complaining of `json` not being defined. I don't understand how `res.json()` can work as this is already done in backendSrv. Might be something old that I can remove? WDYT @grafana/observability-squad

Reply:
I think this was done for testing where `data` would be the response that the test injected, but this might have evolved to no longer being used.

Reply:
Thank you David, I think I'll remove it and see if it explodes in the wild💥